roles/spads/tasks/swap.yaml

```diff
@@ -5,21 +5,31 @@
     get_mime: no
   register: swap_file_check
 
-- name: Create swap file
+- name: Get current swap file size
+  ansible.builtin.command: du -m {{ swap_file }}
+  register: swap_file_size
+  when: swap_file_check.stat.exists
+  changed_when: false
+
+- name: Parse current swap file size
```

Comment:
`ansible.builtin.stat` used at the top returns file size, calling `du` and then parsing output is redundant.